core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -215,6 +219,33 @@ export class ContributionOpportunitiesBackendApiService {
       return [];
     }
   }
+
+  async savePreferredTranslationLanguageAsync(
+      languageCode: string
+  ): Promise<void> {
+    const postData: {'language_code': string} = {
+      language_code: languageCode,
+    };
+    const body = new FormData();
```

Comment:
Do you actually need to create a FormData() here? I looked through the codebase and you can just pass the postData in a dict, I think? (see admin-backend-api.service.ts, playthrough-backend-api.service.ts)

Reply:
I have updated this and now I am just passing the postData in a dict directly.